test/screens/home_calendar_test.dart

```diff
@@ -0,0 +1,31 @@
+
+import 'package:bookoo2/screens/home_calendar.dart' show HomeCalendar;
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import '../test_utils.dart' show TestUtils;
+
+void main() {
+  testWidgets('Show [Ledgers] on the date when the corresponding date has been clicked', (WidgetTester tester) async {
+    await tester.pumpWidget(TestUtils.makeTestableWidget(child: HomeCalendar()));
+    await tester.pumpAndSettle();
+
+    var findByText = find.byType(Text);
+    expect(findByText.evaluate().isEmpty, false);
+
+    expect(find.text('운동'), findsNothing);
```

Comment:
You can just test this with localized param like 
`expect(find.text('EXERCISE'), findsNothing)`

Reply:
Fixed it!